cmd/minikube/cmd/start.go

```diff
@@ -173,6 +173,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		os.Setenv(k, v)
 	}
 
+	//Create the host directories if not present
+	hostDirs := []string{viper.GetString(hostConfigDir), viper.GetString(hostDataDir), viper.GetString(hostVolumesDir)}
+	err = cluster.EnsureHostDirectories(libMachineClient, hostDirs)
```

Comment:
Note: the EnsureDirectories naming is similar to as used in OpenShift, but I had concerns with that in their code, as the intention was only clear after reading the implementation.